components/card.css

```diff
@@ -0,0 +1,126 @@
+
+/* Basic card */
+
+.card {
+    width: 310px;
+    border-radius: 10px;
+    min-height: 430px;
+    height: min-content;
+    display: flex;
+    flex-wrap: wrap;
+    padding-bottom: 10px;
+    overflow: hidden;
+    position: relative;
+  }
+  .card-head {
+    position: relative;
+    width: 100%;
+  }
+  .card-body {
+    padding-left: 24px;
+    padding-right: 24px;
+  }
+  
+  /* Card image */
+  
+  .card-image {
+    border: none;
+    height: min-content;
+    width: 100%;
+    background-color: grey;
+  }
+  .card-image-img {
+      margin-left: auto;
+      margin-right: auto;
+    height: 100%;
+    display: block;
+  }
+  
+  /* card cross */
+  
+  .card-cross {
+      position: absolute;
+      font-size: 2.7rem;
+      color: var(--grey);
+      top: 7px;
+      right: 7px;
+      cursor: pointer;
+    }
+  
+  /* card icon */
+  
+  .card-icon {
+    position: absolute;
+    font-size: 2.7rem;
+    color: var(--grey);
+    bottom: 7px;
+    right: 7px;
+    cursor: pointer;
+  }
+  .card-icon:hover {
+    color: var(--hover-red);
+  }
+  
+  /* card badge */
+  
+  .card-badge {
+    position: absolute;
+    font-size: 1.8rem;
+    top: 10px;
+    padding-left: 3px;
+    padding-right: 3px;
+    border-radius: 3px;
+    background-color: var(--primary);
+    color: white;
+  }
+  
+  /* card text overlay */
+  
+  .card-overlay{
+    opacity: 0.2;
+  }
+  .card-overlay-text{
+      position: absolute;
+      left: 0;
+      bottom: 0;
+      right: 0;
+      top: 0;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+  }
+  .card-overlay-text .textbox{
+      background-color: white;
+  }
+  
+  /* card horizontal */
+  
+  .card-horizontal{
+      padding: 8px;
+      justify-content: left;
+      align-items: center;
+      flex-direction: row;
+      display: flex;
+      min-width:700px;
```

Comment:
have consistent spacing after colon.
At other places you have given one space after colon.